mcs/class/corlib/System.Runtime.InteropServices/Marshal.cs

```diff
@@ -76,6 +77,9 @@ public static bool AreComObjectsAvailableForCleanup ()
 
 		[MethodImplAttribute(MethodImplOptions.InternalCall)]
 		public extern static IntPtr AllocCoTaskMem (int cb);
+		
+		[MethodImplAttribute(MethodImplOptions.InternalCall)]
+		public extern static IntPtr AllocCoTaskMemSize (UIntPtr sizet);
```

Comment:
My bad, just added this one to tell this one apart from the int version, which is limiting by design.